user/deployment/script.md

```diff
@@ -27,3 +27,19 @@ Be sure to make the script executable.
 
 If the script returns a nonzero status, deployment is considered
 a failure, and the build will be marked as "errored".
+
+## Deployment is executed by Ruby 1.9.3
+
+In order to ensure that deployments are executed reliably, we use a version of Ruby that
+is pre-installed on all of our build images.
+Currently, this is Ruby 1.9.3.
+
+A side effect of this is that, if you need to execute a command
+that requires a different Ruby, you need to execute it explicitly.
```

Comment:
@BanzaiMan, may be is also worth to document that if you execute some script at `after_deploy:` the rvm environment will be the same as the one on the build.